### ISSUE TITLE
I have trained the FF++ model and performed incremental training on the DFDC dataset. Is the train_list in TaskN_KD.py referring to the DFDC dataset or the FF++ dataset?

### ISSUE THREAD

**Issue by ats4869** [DeepFakeIL/DFIL#11]
Firstly, you should download all datasets including FF++, DFDC-p, CDF, and DFD.

Secondly, you can use the file 'train_CNN_SupCon_and_CE.py' to train your first detection model with the FF++ dataset.

Thirdly, you can use 'get_feature.py', 'get_image_info.py', and 'create_memory.py' to construct your memory set.

Finally, I don't quite understand the part about randomly picking 25 training videos from your new dataset and adding them to your memory set to train a new model using the file 'TaskN_KD.py'. Are the 25 training videos in the `train_list`?

Please explain how to fill in these hyperparameters:

parse = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
parse.add_argument('--name', '-n', type=str, default='20230425_Task2_DFDCP_with_FD')

parse.add_argument('--train_list', '-tl', type=str, default='2500_real_and_2500_fake_train.txt')

parse.add_argument('--memory_list', '-ml', type=str, default='2020230424_Task2_DFDC_with_1consup_and_01_fd_Memory_DFDCP_img_info_Memory copy.txt')

parse.add_argument('--val_list', '-vl', type=str, default='2500_real_and_2500_fake_train.txt')

parse.add_argument('--batch_size', '-bz', type=int, default=64)

parse.add_argument('--epoches', '-e', type=int, default='10')

parse.add_argument('--model_name', '-mn', type=str, default='demo.pkl')

parse.add_argument('--continue_train', type=bool, default=True)

parse.add_argument('--add_memory', type=bool, default=True)

parse.add_argument('--model_path', '-mp', type=str, default='20230424_Task2_DFDC_with_1consup_and_01_fd/best.pkl')

parse.add_argument('--teacher_model_path', '-tmp', type=str, default='20230424_Task2_DFDC_with_1consup_and_01_fd/best.pkl')
```

In the context of performing incremental training on the DFDC dataset, which parameters should be associated with the DFDC dataset, particularly `train_list`, `val_list`, and `memory_list`?

**Comment by DeepFakeIL** [comment]
--name: this is your file name to save trained model;
--train_list: this is your train list inclduing image path and label;
--memory_list: this is your memory list to save hard samples including  image path and label;
--val_list: this is your val list inclduing image path and label;
--batch_size: pass
--epoches: pass
--model_name: pass
--continue_train:pass
--add_memory: this parameter controls if you need to use memory set. eg. task 1 set to false; task 2,3,4,5.... set to ture;
